### PR TITLE
add microsoft gem

### PR DESCRIPTION
**Pull request by antopalidi**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -21,6 +21,8 @@ gem "faker", "~> 2.14"
 
 gem "wicked_pdf", "~> 2.1"
 
+gem 'decidim-microsoft'
```
Comment (microstudi) [inline Gemfile:24]:
```suggestion
gem 'decidim-microsoft', "~ 1.0.0"
```

Why don't we specify the version number here?

**Review by microstudi — CHANGES_REQUESTED** [review]
Looks very nice but I left you a comment

File: Gemfile (hunk 1 above)
Comment (antopalidi) [reply Gemfile:24]: ok, I will do it

**Review by microstudi — APPROVED** [review]
thanks!

**Pull request merged**